logging/tests/system.py

```diff
@@ -171,6 +171,24 @@ def test_log_text_with_timestamp(self):
         self.assertEqual(entries[0].payload, text_payload)
         self.assertEqual(entries[0].timestamp, now.replace(tzinfo=UTC))
 
+    def test_log_text_with_resource(self):
+        from google.cloud.logging.resource import Resource
+        text_payload = 'System test: test_log_text_with_timestamp'
+
+        logger = Config.CLIENT.logger(self._logger_name())
+        now = datetime.datetime.utcnow()
+        RESOURCE = Resource(
+            type='gae_app', labels={'module_id': 'default',
+                                    'version_id': 'test'})
```

Comment:
This is a style nit, so feel free to ignore, but I would prefer:

```python
resource = Resource(
    type='gae_app',
    labels={
        'module_id': 'default',
        'version_id': 'test',
    },
)
```

(Also you don't need to / shouldn't treat `RESOURCE` like a constant. I realize our unit tests do that, but it's "not a thing".)